Document/0x04f-Testing-Network-Communication.md

```diff
@@ -203,165 +167,128 @@ If a mobile application connects to a specific server, its networking stack can
 - [testssl.sh](https://github.com/drwetter/testssl.sh) which "is a free command line tool which checks a server's service on any port for the support of TLS/SSL ciphers, protocols as well as some cryptographic flaws".
 
-
 ### Making Sure that Critical Operations Use Secure Communication Channels
 
 #### Overview
 
-For sensitive applications like banking apps, [OWASP MASVS](https://github.com/OWASP/owasp-masvs/blob/master/Document/0x03-Using_the_MASVS.md "The Mobile Application Security Verification Standard") introduces "Defense in Depth" verification levels. The critical operations (e.g., user enrollment and account recovery) of such applications are some of the most attractive targets to attackers. This requires implementation of advanced security controls, such as additional channels (e.g., SMS and e-mail) to confirm user actions.
+For sensitive applications like banking apps, [OWASP MASVS](https://github.com/OWASP/owasp-masvs/blob/master/Document/0x03-Using_the_MASVS.md "The Mobile Application Security Verification Standard") introduces "Defense in Depth" verification levels. The critical operations (e.g., user enrollment and account recovery) of such applications are some of the most attractive targets to attackers. This requires implementation of advanced security controls, such as additional channels besides email and/or SMS to confirm user actions.
+
+Note that using SMS as an additional factor for critical operations is not recommended. Attacks like SIM swap scams were used in many cases to attack Instagram accounts, financial institutions and cryptocurrency exchanges to bypass SMS verification. SIM swapping is a legitimate service offered by many carriers to switch your mobile number to a new SIM card. If an attacker manages to either convince the carrier or recruits retail workers at mobile shops to do a SIM swap, the mobile number will be transferred to a SIM the attacker owns.  As a result of this, the attacker will be able to receive all SMS and voice calls without the victim knowing it.
+
+There are different ways to [protect your SIM card](https://www.wired.com/story/sim-swap-attack-defend-phone/ "How to protect yourself against a SIM swap attack"), but this level of security maturity and awareness cannot be expected from a normal user and is also not enforced by the carriers.  
+
+Also the usage of emails shouldn't be considered as a secure communication channel.
 
 #### Static Analysis
 
 Review the code and identify the parts that refer to critical operations. Make sure that additional channels are used for such operation. The following are examples of additional verification channels:
 
-- Token (e.g., RSA token, yubikey);
-- Push notification (e.g., Google Prompt);
-- SMS;
-- E-mail;
-- Data from another website you visited or scanned;
+- Token (e.g., RSA token, YubiKey),
+- Push notification (e.g., Google Prompt),
+- Data from another website you visited or scanned or
```

Comment:
Part missing?

Reply:
Actually these are just examples. But what do you think about this to make the scanning more clear: 

> Review the code and identify the parts that refer to critical operations. Make sure that additional channels are used for such operations. The following are examples of additional verification channels:
>- Token (e.g., RSA token, YubiKey),
>- Push notification (e.g., Google Prompt),
>- Data from another website you have visited or scanned (e.g. QR code) or
>- Data from a physical letter or physical entry point (e.g., data you receive only after signing a document at a bank).